### ISSUE TITLE
I'm sorry to bother you,but I have some questions

### ISSUE THREAD

**Issue by erhuodaosi** [SeitaroShinagawa/chainer-partial_convolution_image_inpainting#27]
First of all,given an arbitrary broken picture, how to get its mask area?
Furthermore,how to test on a real broken image?I have used your ways to retrain the model but in vain,the generated results still have the broken part.
Last but not the least,what is the meaing of using VGG model,pretraining or caculating the function loss ?
I will very appreciate if you could help me,thank you!

**Comment by SeitaroShinagawa** [comment]
Actually, the mask area must be given in advance because this task focuses on image editing, not image restoration.  If you create a mask manually, this model can in-painting the masked area.   

The purpose of using VGG pre-trained model is to calculate loss function. You are correct.  

Best,

**Comment by erhuodaosi** [comment]
I have understood your meaing,thank you for your imediate reply!
